<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="openFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveConfigurationButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setDefaultConfig">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set the default config</string>
   </property>
  </widget>
  <widget class="QListView" name="dataListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>211</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="chartsListView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>261</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addChartButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Chart</string>
   </property>
  </widget>
  <widget class="QListView" name="dataChartListView">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>291</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addDataButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add data to chart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showChartsButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>240</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Charts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteChartButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Chart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeDataButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove data from chart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openConfigurationButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearDefaultConfig">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear the default config</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
